store/repository.go

```diff
@@ -60,12 +60,6 @@ func NewRepository(store *Store) *Repository {
 
 func (e *Repository) UpdateOrCreateEntity(item proto.Entity) (string, error) {
 
-	hash, err := hash(item)
-
-	if err != nil {
-		return "", err
-	}
-
 	var bucket bucket
```

Comment:
I've been thinking about this, though Go doesn't have a clear convention about new types names, I think for clarity sake, we should establish a convention to distinguish them from other identifiers.
What do you think about it?